docs/src/pages/guides/FHIRPerformanceGuide.md

```diff
@@ -453,15 +453,15 @@ Derby is not recommended for production use and therefore tuning Derby will not
 
 ## 5.1. Logical Id Generation
 
-Using random values for resource identifiers can cause performance issues in large databases. This is a particular issue when using PostgreSQL with the IBM FHIR Server due to an issue known as write amplification from full page writes. For details, see this blog post: https://www.2ndquadrant.com/en/blog/on-the-impact-of-full-page-writes.
+Using random values for resource identifiers can cause performance issues in large databases. This is a particular issue when using PostgreSQL with the LinuxForHealth FHIR Server due to an issue known as write amplification from full page writes. For details, see this blog post: https://www.2ndquadrant.com/en/blog/on-the-impact-of-full-page-writes.
 
 For best performance, ids generated by clients should not be purely random but instead be structured to include a prefix which increments over time. This causes index entries for new values to share pages (right-hand inserts), greatly reducing the write amplification overhead.
 
 One example of a suitable id generation strategy can be found in the [IBM FHIR Server fhir-persistence-jdbc project](https://github.com/LinuxForHealth/FHIR/blob/main/fhir-persistence-jdbc/src/main/java/org/linuxforhealth/fhir/persistence/jdbc/util/TimestampPrefixedUUID.java).
 
 This strategy provides both the desirable trait of global uniqueness as well as a low write amplification overhead thanks to the time-based prefix.
 
-The IBM FHIR Server also uses normalization to avoid storing (and indexing) long identifier strings in multiple places. This saves space, and the database-generated identity values are based on sequences which naturally produce the desired right-hand-insert behavior.
+the LinuxForHealth FHIR Server also uses normalization to avoid storing (and indexing) long identifier strings in multiple places. This saves space, and the database-generated identity values are based on sequences which naturally produce the desired right-hand-insert behavior.
```

Comment:
```suggestion
The LinuxForHealth FHIR Server also uses normalization to avoid storing (and indexing) long identifier strings in multiple places. This saves space, and the database-generated identity values are based on sequences which naturally produce the desired right-hand-insert behavior.
```